readme.md

```diff
@@ -279,6 +279,7 @@
 | [Font Awesome](https://fontawesome.com/)| The web's most popular icon set and toolkit  |
 | [Line Awesome](https://icons8.com/line-awesome)| Swap in replacement of Font Awesome with modern line icons |
 | [Material Icons](https://material.io/resources/icons/)| Material design icon library |
+| [Fluent UI Icons](https://developer.microsoft.com/en-us/fluentui#/styles/web/icons)| Fluent UI icons by Microsoft. [How to set it up?](https://blog.usman-s.me/how-to-use-microsoft-fluent-ui-icons-on-your-website) |
```

Comment:
Please move it to react section as it's react UI.

Also please remove the how to setup link.

Thanks

Reply:
Not specifically React because it can be used in static sites with a CDN and in React with it's package